projection/core/src/multi-jvm/scala/com/lightbend/lagom/internal/projection/ProjectionRegistrySpec.scala

```diff
@@ -101,6 +118,22 @@ class ProjectionRegistrySpec extends ClusteredMultiNodeUtils(numOfNodes = 3) wit
       expectProjectionStatus(projectionName, 5, Stopped)
     }
 
+    "request the pause of a complete projection (before projection is registered)" in {
+      enterBarrier("request-pause-projection-test-before-registering")
+      val projectionName = "test-pause-projection-before-registering"
+      val tagNamePrefix  = projectionName
+      val tagNames       = (1 to 5).map(id => s"$tagNamePrefix-$id")
+
+      runOn(RoleName("node2")) {
+        projectionRegistry.stopAllWorkers(projectionName)
+      }
+
+      registerProjection(projectionName, tagNames.toSet)
+      enterBarrier("sync-request-pause-projection-test-before-registering")
+
+      expectProjectionStatus(projectionName, 5, Stopped)
+    }
```

Comment:
These two new tests demonstrate requests arriving before a projection is even registered is now safe and honored.

Reply:
... before this PR, the requests were silently discarded.